pkgs/by-name/sd/SDL_compat/package.nix

```diff
@@ -50,6 +50,11 @@ stdenv.mkDerivation (finalAttrs: {
 
   setupHook = ./setup-hook.sh;
 
+  postInstall = ''
+    # allow as a drop in replacement for SDL
```

Comment:
Can you maybe add a comment saying we can remove this when we switch to pkgconf?

Reply:
Done.